consensus/src/marshal/actor.rs

```diff
@@ -102,9 +107,15 @@ pub struct Actor<B: Block, E: Rng + Spawner + Metrics + Clock + GClock + Storage
     processed_height: Gauge,
 }
 
-impl<B: Block, E: Rng + Spawner + Metrics + Clock + GClock + Storage, V: Variant> Actor<B, E, V> {
+impl<
+        E: Rng + CryptoRng + Spawner + Metrics + Clock + GClock + Storage,
+        B: Block,
+        Pr: SigningSchemeProvider<S>,
```

Comment:
abbreviation consistency: `Pr` here but `P` up above.

Reply:
The reason is that there's some methods in the impl that already take `P`. I can change the type name on the methods though.